CONTRIBUTING.md

```diff
@@ -83,13 +101,50 @@ npm run dev:pristine
+1. Use the `Sky` prefix when naming all classes, directives, services, components, etc. The prefix indicates to other contributors that items are owned by SKY UX and not a third-party library. It also prevents potential class-name clashes with other libraries. Keep in mind that while we generally use the uppercase `Sky` prefix, we also use the `sky-` prefix in some cases, such as selector properties in components.
+
+## Test harnesses
+
+When you create components, include a test harness so that consumers can use them in unit tests to query against the components. These harnesses should be placed in the library's `testing` folder and should be exported via the `testing/public-api.ts` file. These harnesses should extend the `SkyComponentHarness` class which extends the Angular CDK `ComponentHarness` class.
+
+The test harness should give consumers the ability to query the harness for information about the component which may rely on the component's internal structure to determine. This allows consumers to avoid writing brittle tests that may break due to internal changes within a major version.
+
+All items in the `testing` folder should be fully unit tested with same standards which apply to the main component library.
+
+## Unit tests
+
+### Code coverage
+
+All new code requires 100 percent unit test code coverage. This doesn't guarantee that every use case is accounted for, but anything less than 100 percent code coverage guarantees that at least one use case is not accounted for. This can be verified by running tests with npm run watch and viewing the code coverage results in `./coverage/index.html`. You can launch this straight from disk and view the SKY UX unit test code coverage results in your default web browser.
+
+### Naming convention
+
+Unit tests should follow the Angular guidelines for naming and storing spec files. Name tests after the components or services that they test, and place them in the same folders as their components or services. For example, the test file for `foo/foo.component.ts` should be `foo/foo.component.spec.ts`.
+
+### Run tests
+
+To run unit tests from the command line, use the `npm run test:affected` command. To run unit tests and watch the file system for changes, use the command `npx nx test <library> --watch`. SKY UX styles are loaded when running unit tests, so CSS rules defined in SKY UX take effect during tests. This allows you to check the expected computed style of elements with properties, such as the HTML hidden property, that only take effect when SKY UX styles are loaded.
+
+## Accessibility tests
+
+All new components and changes to existing components require passing accessibility tests. This includes automated tests to run during unit tests as well as code review and manual keyboard tests. For more information about accessibility tests, see the [accessibility section of the SKY UX documentation](https://developer.blackbaud.com/skyux/learn/accessibility).
+
+## Visual regression tests
```

Comment:
I don't see this section anywhere in the docs site. Do we need to add it? I could have sworn that we added something about visual regression tests at some point, but I'm not sure if I'm misremembering or if maybe someone removed it.

Reply:
We had the section - but it was removed when we didn't have visual tests for a while. This language was from the historical language that we had and then I tweaked it a little for our current setup.

The docs pages are going away - so we don't need to update that.